acme-cli/src/test/groovy/io/micronaut/acme/cli/AcmeCliCommandSpec.groovy

```diff
@@ -0,0 +1,30 @@
+package io.micronaut.acme.cli
+
+import io.micronaut.acme.cli.AcmeCliCommand
+import io.micronaut.configuration.picocli.PicocliRunner
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.env.Environment
+
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+class AcmeCliCommandSpec extends Specification {
+
+    @Shared @AutoCleanup ApplicationContext ctx = ApplicationContext.run(Environment.CLI, Environment.TEST)
+
+    void "acme-cli with no subcommand just returns the help text"() {
+        given:
+        ByteArrayOutputStream baos = new ByteArrayOutputStream()
+        System.setErr(new PrintStream(baos))
+
+        PicocliRunner.run(AcmeCliCommand, ctx)
+
+        expect:
+        baos.toString().contains("Please invoke a subcommand")
+        baos.toString().contains("Usage: acme-cli [COMMAND]")
```

Comment:
Test is currently failing, works with this:
```suggestion
        baos.toString().contains("Usage: acme-cli [-h] [COMMAND]")
```

Reply:
Fixed here c51179e